lib/stack_master/parameter_resolvers/one_password.rb

```diff
@@ -20,25 +22,22 @@ def resolve(params={})
       def validate_op_installed?
         %x(op --version)
         rescue Errno::ENOENT => exception
-          raise RuntimeError, "The op cli needs to be installed and in the PATH, #{exception}"
+          raise OnePasswordBinaryNotFound, "The op cli needs to be installed and in the PATH, #{exception}"
       end
 
-      def parseable_json?(item)
+      def validate_response?(item)
+        item.match(/\[LOG\].+(?<error>\(.+)$/)  do |i|
```

Comment:
What happens if the output is something other than an `ERROR`?

Reply:
It will be picked up as invalid JSON. An error appears to have a `[LOG]` prefix if it gets an error response from `op` so it should be picked up here, otherwise it should have invalid JSON and be picked up there. 

```
Please enter your password for user denmat:
[LOG] 2018/04/16 15:00:18 (ERROR) 401: Authentication required.
```